src/buildercore/bootstrap.py

```diff
@@ -19,6 +19,7 @@
 from boto.exception import BotoServerError
 from kids.cache import cache as cached
 from buildercore import cfngen
+import buildvars
```

Comment:
is this the `buildvars.py` from `src/` ? if so, it doesn't belong here. refactoring is needed. 
nothing inside `buildercore` relies on the interface code in it's parent.